internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,60 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+
+	stringsplit := strings.Split(datastring, ",")
+
+	if len(stringsplit) != 2 {
+		return fmt.Errorf("ошибка формата данных")
+	}
+
+	steps, err1 := strconv.Atoi(stringsplit[0])
```

Comment:
Ошибки можно не плодить, а использовать одну переменную для всех
```suggestion
	steps, err := strconv.Atoi(stringsplit[0])
```